apps/functions-subscription/src/adapters/azure/functions/__tests__/list-trials.test.ts

```diff
@@ -0,0 +1,144 @@
+import { describe, expect, it } from 'vitest';
+import * as TE from 'fp-ts/TaskEither';
+import { makeAValidListTrialRequest, supportHttpRequestHeaders } from './data';
+import { makeFunctionContext, makeTestSystemEnv } from './mocks';
+import { makeListTrialsHandler } from '../list-trials';
+import { aTrial } from '../../../../domain/__tests__/data';
+import { HttpRequest } from '@azure/functions';
+import { Trial, TrialId } from '../../../../domain/trial';
+
+describe('makeListTrialsHandler', () => {
+  const anotherTrial = {
+    ...aTrial,
+    id: 'anotherTrialId' as TrialId,
+  } as Trial;
```

Comment:
**suggestion**: same as previous commend, unnecessary cast

Reply:
Removed in 135e4c218e4d43aeae3ea6cb5dff5f2bf85a5fe9